tailwind.config.ts

```diff
@@ -14,6 +14,60 @@ const config: Config = {
           'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
       },
     },
+    fontFamily: {
+      serif: ['DM Serif Display', 'serif'],
+      sans: ['DM Sans', 'sans-serif'],
+    },
+    fontWeight: {
+      regular: '400',
+      medium: '500',
+      bold: '700',
+    },
+    fontSize: {
+      display: ['64px', { lineHeight: '72px' }],
+      h1: ['56px', { lineHeight: '64px' }],
+      h2: ['48px', { lineHeight: '56px' }],
+      h3: ['40px', { lineHeight: '48px' }],
+      h4: ['32px', { lineHeight: '40px' }],
+      h5: ['24px', { lineHeight: '32px' }],
+      h6: ['20px', { lineHeight: '28px' }],
+      p: ['16px', { lineHeight: '24px' }],
+      label: ['14px', { lineHeight: '22px' }],
+      tiny: ['12px', { lineHeight: '20px' }],
```

Comment:
https://github.com/isac-nitkkr/node-modules/issues/50#issuecomment-1917598888

Reply:
```ts
    fontSize: {
      '6xl': ['64px', { lineHeight: '72px' }],
      '5xl': ['56px', { lineHeight: '64px' }],
      '4xl': ['48px', { lineHeight: '56px' }],
      '3xl': ['40px', { lineHeight: '48px' }],
      '2xl': ['32px', { lineHeight: '40px' }],
      xl: ['24px', { lineHeight: '32px' }],
      lg: ['20px', { lineHeight: '28px' }],
      base: ['16px', { lineHeight: '24px' }],
      sm: ['14px', { lineHeight: '22px' }],
      xs: ['12px', { lineHeight: '20px' }],
    },
```

Is this fine? The labels are from [TailwindCSS docs](https://tailwindcss.com/docs/font-size)

Reply:
This is good. Can you make `xs`, etc. also inside quotes for consistency?

Reply:
Okay

Reply:
This will create warnings from prettier. Should I put it under prettier-ignore or keep as it is?

Reply:
Keep it as is then.

Reply:
Pushed the changes.